selenium_tests/base.py

```diff
@@ -1,4 +1,9 @@
-"""Basic selenium tests for MicroMasters"""
+"""
+NOTE: THIS MODULE IS DEPRECATED AND SHOULD BE REMOVED ONCE THE 'snapshot_dashboard_states' COMMAND IS
```

Comment:
kind of difficult to say, to be honest. i hadn't seen that module before this PR and there's a lot going on. i agree that it's not a great thing to keep this around. if you're comfortable with the increased scope of the PR, i'm happy to work on that and push a fix